include/benchmark/benchmark.h

```diff
@@ -1295,7 +1295,7 @@ struct CPUInfo {
     int num_sharing;
   };
 
-  enum class Scaling {
+  enum Scaling {
```

Comment:
ah yes, because it's in the header. :)

Reply:
Any insights? :) Is C++03 used at present in 2020? :)

Reply:
yes. we still support it for installations. ie, you need >c++11 to _build_ the library, but you shouldn't need it to link to the library. this will change once the base libc/compilers change for LTS OSes.